pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -138,7 +138,7 @@ func (m *managerImpl) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAd
 		return lifecycle.PodAdmitResult{Admit: true}
 	}
 	// the node has memory pressure, admit if not best-effort
-	if hasNodeCondition(m.nodeConditions, v1.NodeMemoryPressure) {
+	if hasNodeCondition(m.nodeConditions, v1.NodeMemoryPressure) && len(m.nodeConditions) == 1 {
```

Comment:
```suggestion
        nodeOnlyHasMemoryPressureCondition := hasNodeCondition(m.nodeConditions, v1.NodeMemoryPressure) && len(m.nodeConditions) == 1
	if nodeOnlyHasMemoryPressureCondition {
```

Thoughts on the above small modification? In my mind, it has the advantage of replacing a boolean expression/comment with an explanatory variable name.

Reply:
Good suggestion :) , this code style is quite common in k8s code, thanks!

Reply:
Hmm... Never thought a committed suggestion would create a new commit, I'll merge them into one, thank you again :)